api/openapi.yaml

```diff
@@ -56,6 +56,34 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/ApiItem"
+    post:
+      tags:
+        - "apis"
+      summary: "Deploy new API"
+      description: "Deploys a new API to the cluster"
+      operationId: deployApi
+      requestBody:
+        description: "API content that needs to be deployed"
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+      responses:
+        201:
+          description: "API deployed"
+        400:
+          description: "The content of the API is malformed"
+          content:
+            application/json:
+              schema:
+                type: object
```

Comment:
maybe 

`type: string` 

is enough to start with? @jasmingacic what do you think?

Reply:
It should be enough